libexec/mcd/mcd

```diff
@@ -122,16 +125,16 @@ mcd-init() {
     *)
       if [[ "$@" != *"help"* ]]; then
         echo "Warning: \`--chain' option not specified. Defaulting to kovan"
-        config-init ${0%/*}/conf/kovan.json;
+        config-init "${0%/*}/conf/kovan.json";
         export SETH_CHAIN=kovan
       fi
   esac
 }
 
 if ! [[ $MCD_INIT ]]; then
-  TESTNET=${TESTNET:-~/.dapp/testnet}
+  TESTNET="${TESTNET:-~/.dapp/testnet}"
   export MCD_INIT=1
-  mcd-init
+  mcd-init 1
```

Comment:
What is the purpose of adding "1"? Something related to shellcheck?

Reply:
That's a good question. It looked like it was setting and reinitializing the testnet here, so adding the `1` sets the other vars to testnet on the reinit. I did some testing and it's unnecessary, removed at 6683d2a3cce30ebae98a4b73ad00ba0c85acb4ef